go.mod

```diff
@@ -57,3 +57,5 @@ require (
 )
 
 go 1.14
+
+replace github.com/lucas-clemente/quic-go v0.20.1 => github.com/lucas-clemente/quic-go v0.19.3
```

Comment:
wat

Reply:
pfilter uses quic in tests/benchmarks. The version I use is newer than the one we use here, so pulling in pfilter updates quic-go. I'm forcing it back to the original version we had.

Reply:
Does pfilter require the new features in QUIC .20? It seems like it should depend on the older version otherwise, for specifically this reason...

Reply:
It depends on it purely for the purpose of tests, but go.mod does not differentiate between test and non-test dependencies.

Reply:
Well this "we depend on quic-go 0.20.1" and later "haha just kidding when I said 0.20.1 I mean 0.19.3" is really weird... For better or worse Go does the MVS thing and so ideally dependencies that don't care about the specifics should depend on some lower version and let the main module push it upwards, instead of the other way around...